e2e/test/scenarios/embedding/embedding-linked-filters.cy.spec.js

```diff
@@ -384,9 +384,10 @@ describe("scenarios > embedding > dashboard > linked filters (metabase#13639, me
           .and("contain", "Doohickey");
 
         cy.log("Make sure we can set multiple values");
-        visitEmbeddedPage(payload, {
-          setFilters: "id_filter=4&id_filter=29&category=Widget",
-        });
+        cy.window().then(
+          win =>
+            (win.location.search = "?id_filter=4&id_filter=29&category=Widget"),
+        );
```

Comment:
This is a workaround for a particularly weird implementation of ours.
We set multiple filter values by repeating the filter name. It's not possible to create an object with multiple same properties in order to pass it to `new URLSearchParams()`, so we'll have to manually update `window.location.search` in such cases.

Reply:
could we change the api to use an array of tuples (`[key, value]`) instead of a map? e.g

```js
visitEmbeddedPage(payload, { setFilters: [['id_filter', 4], ['id_filter', 5]] });

```